examples/time-log-kopia-repository-server/README.md

```diff
@@ -0,0 +1,231 @@
+# Protecting TimeLog application
+
+This is an example of using Kanister to protect a simple TimeLog application. This application is contrived but useful for demonstrating Kanister's features. Every second it appends the current time to a log file.
+
+Note: This application is used in the Kanister tutorial, for a detailed walkthrough click [here](https://docs.kanister.io/tutorial.html#tutorial).
+
+### 1. Deploy the Application
+
+The following command deploys the example Time Log application in `time-log` namespace:
+
+```bash
+$ kubectl create namespace time-log
+
+# Create a deployment whose log we'll ship to s3 using Kanister functions (that would eventually use Kopia repository server)
+$ kubectl apply -f ./examples/time-log/time-logger-deployment.yaml -n time-log
+deployment "time-logger" created
+
+### 2. Protect the Application
+
+#### Create Repository Server CR
+
+Create Kopia Repository Server CR if not created already
```

Comment:
This can be confusing to user, since we are creating the repository server CR at https://github.com/kanisterio/kanister/pull/2094/files#diff-8dd680206f26a3173794379cb22c1dc99b1445431896ec185c501c6c4ec4d72cR118

Reply:
lets remove the line

Reply:
Done !